README.md

```diff
@@ -1 +1,72 @@
-# .github
\ No newline at end of file
+
+## Repositories: 
+- Service
+- Purpose
+- Language
+- External Integration
+- Database
+- Infrastructure
+- Main contributor
+
+### 1. ubiquibot/conversation-rewards
+- Service: Conversation Reward Module
+- Purpuse: This is intended to be the proper implementation of comment incentives, based on our learnings from the first go-around.
+- Language: TypeScript
+- External Integration: Github Action, Github API
+- Database: None
+- Main contributor: @pavlovcik
+
+
+### 2. ubiquibot/permit-generation
+- Service: Permit Generation module
+- Purpuse: The purpose of this module is to isolate the bot's payment capabilities. This can be directly invoked from other plugins
+- Language: TypeScript
+- External Integration: ENS, ethers
```

Comment:
```suggestion
- External Integration: ENS
```

Not sure if I would consider `ethers` "external" but it also is worth discussing what technically is considered "external."